recipes/main/schaeufele.adoc

```diff
@@ -19,23 +19,23 @@ the pan of a shovel and comes from the shape of the bone (pork shoulder).
 
 ==== Ingredients
 
-[width="80%",cols="2,2,10",options="header"]
+[width="100%",cols="3,3,10,10",options="header"]
 |=========================================================
-| English | Metric | Ingredient 
-
-| 3-4 lb | 1,5-2 kg | Pork shoulder with bone and rind
-| | | Salt 
-| | | Black pepper (freshly grounded)
-| | | Caraway (grounded)
-| | | Butter oil / resolidified butter 
-| 2-3 | 2-3 | Onions 
-| 1 | 1 | Leek 
-| 2-3 | 2-3 | Carrots 
-| 1/2 | 1/2 | Celery root
-| 2 | 2 | Garlic cloves
-| | | Parsley (twigs)
-| 4 cups | 1 l | Broth
-| 7 fl oz | 200 ml | Beer (e.g. lager, brown ale, not hoppy)
+| US | Metric | Ingredient | Zutat
+
+| 3-4 lb | 1,5-2 kg | Pork shoulder with bone and rind | Schäufele™
+| | | Salt | Salz
+| | | Black pepper (freshly grounded) | Schwarzer Pfeffer (frisch gemahlen)
+| | | Caraway (grounded) | Kümmel (gemahlen)
```

Comment:
```suggestion
| | | Caraway seeds (grounded) | Kümmel (gemahlen)
```

Reply:
Thanks for spotting these, fixed in latest commit.